pkg/codegen/go/gen_program.go

```diff
@@ -584,7 +587,64 @@ func (g *generator) genLocalVariable(w io.Writer, v *hcl2.LocalVariable) {
 		g.Fgenf(w, "%s := %.3v;\n", name, expr)
 
 	}
+}
+
+func (g *generator) genConfigVariable(w io.Writer, v *hcl2.ConfigVariable) {
+	if !g.configCreated {
+		g.Fprintf(w, "%sconfig := config.New(ctx, \"\")\n", g.Indent)
```

Comment:
I think we need to use a different name here (can't reassign `config`). Also, given that the generator runs `go fmt` at the end, we don't need to worry about indentation (applies elsewhere as well).

Reply:
Done